modularhistory/config/caches.py

```diff
@@ -10,24 +10,46 @@
 Cache = Dict[str, Any]
 CACHES: Dict[str, Cache]
 
+REDIS_CACHE = {
+    'BACKEND': 'django_redis.cache.RedisCache',
+    'LOCATION': f'{REDIS_BASE_URL}/0',
+    'OPTIONS': {
+        'CLIENT_CLASS': 'django_redis.client.DefaultClient',
+    },
+}
+
 # https://github.com/jazzband/django-redis
 if IS_DEV and use_dummy_cache:
     CACHES = {
         'default': {
             'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
-        }
+        },
+        'cachalot': REDIS_CACHE,
```

Comment:
Was this just for testing?  Do we want to keep it?

Reply:
I left it so it can be enabled easily in dev mode by changing `CACHALOT_CACHE`

Reply:
it should probably need to be improved to respect `use_dummy_cache`